src/ComScoreAnalytics.ts

```diff
@@ -6,14 +6,16 @@ export class ComScoreAnalytics {
   private static started: boolean = false;
   private static configuration: ComScoreConfiguration;
   private static logger: ComScoreLogger;
+  public configuration: ComScoreConfiguration;
```

Comment:
Why do we need this instance variable now? Everything I see in this class is static so there will never be an instance.